dev/com.ibm.websphere.appserver.features/visibility/public/facesContainer-4.0/io.openliberty.facesContainer-4.0.feature

```diff
@@ -17,7 +17,7 @@ IBM-API-Package: org.jboss.weld;type="internal",\
   com.ibm.websphere.appserver.servlet-6.0, \
   com.ibm.websphere.appserver.eeCompatible-10.0, \
   io.openliberty.jakarta.validation-3.0, \
-  io.openliberty.pages-3.1, \
+  io.openliberty.expressionLanguage-5.0, \
   com.ibm.websphere.appserver.jndi-1.0, \
```

Comment:
pages-3.1 pulls in expressionLanguage-5.0 so we need to explicitly pull in expressionLanguage-5.0 here.